css.md

```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
 - Colors: (use generic variable names like recent cfgov-refresh update)
 - Breakpoints: standard variable names for commonly used breakpoints
 
+### Comments
+
+### JSDocs
+[Use JSDocs](http://usejsdoc.org/) style comments for comment blocks and `//` for inline comments.
+
+**Example**
+
+```less
+/**
+ * Default button styles
+ * @class .btn
+ * @elements a, button
+*/
+.btn {
+    display: inline-block;
+
+  &:focus {
+    background-color: @btn-bg-hover;
+    outline: 1px dotted @btn-bg;
+    // outline-offset is not supported everywhere but it adds a nice touch where it is
+    outline-offset: 1px;
+  }
+}
+```
+
+
+Prefer
+
+### Purpose
+Comments **aren't meant to explain what** the code does. Good **code is supposed to be self-explanatory**. If you're thinking of writing a comment to explain what a piece of code does, chances are you need to change the code itself. The exception to that rule is explaining what a regular expression does. Good comments are supposed to **explain why** code does something that may not seem to have a clear-cut purpose.
+
```

Comment:
I think a lot of that statement is still relevant to commenting on Less. Sure you don't want to retain some of it?